strapi/strapi
# UploadDialog: allow removing assets before uploading

### What does it do?

It allows removing assets, after there were selected for uploading but before actually uploading them.

![Screenshot from 2022-02-07 22-09-57](https://user-images.githubusercontent.com/2244375/152872431-3879b96d-bbf3-41e7-ba39-475315d61c9a.png)


### Why is it needed?

It allows users to review the selected assets before uploading them.

### How to test it?

1. Go to the media library
2. Add new assets
3. Select files from your computer
4. You should now be able to edit/ remove assets

### Related issue(s)/PR(s)

- https://strapi-inc.atlassian.net/browse/CONTENT-62
